app/assets/stylesheets/tables-report.css.scss

```
@@ -8,6 +8,26 @@
 @forward 'usa-table';
 @forward 'usa-alert';
 
+hr {
+  background-color: $theme-color-secondary;
+}
+
+.margin-bottom-4 {
+  @include u-margin-bottom(4);
+}
+
+.margin-top-4 {
+  @include u-margin-top(4);
+}
+
+.height-05 {
+  @include u-height(4px);
```

Couple nitpicks:

- I'm not sure we can trust that `$theme-color-secondary` will always be defined, probably safer and simpler to use the `color` utility.
- The `05` in `height-05` refers to its units `0.5`, so we should probably pass that to the utility for consistency. It shouldn't make any effective difference.

```suggestion
  background-color: color('secondary');
}

.margin-bottom-4 {
  @include u-margin-bottom(4);
}

.margin-top-4 {
  @include u-margin-top(4);
}

.height-05 {
  @include u-height(0.5);
```